pkg/services/featuremgmt/registry.go

```diff
@@ -1796,6 +1796,15 @@ var (
 			HideFromAdminPage: true,
 			HideFromDocs:      true,
 		},
+		{
+			Name:              "improveRendererTraffic",
```

Comment:
Are you thinking of reuse this feature flag to turn on/off other improvements? If not, I would change the name and description to something more meaningful like `disableAppPluginsForRender` (or something like that)

Reply:
Yes, I was thinking about reusing it if we have other ideas to improve this but, currently, I have no other idea in mind.

Reply:
My only concern is that when reading feature flags, I like to understand what they do at first glance. If I have a problem preloading apps and I'm investigating the cause, reading an FF like this one won't give me a clue.

Reply:
True! Plus, the name I chose was more user-oriented (what will the user gain to turn this on) but this feature flag will most likely remains internal and hidden from users so let's make our lives easier with a name that will help with debugging, updated :+1: